community/i3-gaps/depends

```diff
@@ -0,0 +1 @@
+../i3/depends
```

Comment:
This file is also not valid, it should contain the proper dependencies of the application.